research/minigo/minigo.py

```diff
@@ -124,19 +106,21 @@ def _write_sgf_data(dir_sgf, use_comments):
         os.path.join(dir_sgf, '{}.sgf'.format(output_name)), 'w') as f:
       f.write(player.to_sgf(use_comments=use_comments))
 
-  _write_sgf_data(clean_sgf, use_comments=False)
-  _write_sgf_data(full_sgf, use_comments=True)
+  _write_sgf_data(selfplay_dirs['clean_sgf'], use_comments=False)
+  _write_sgf_data(selfplay_dirs['full_sgf'], use_comments=True)
 
   game_data = player.extract_data()
   tf_examples = preprocessing.make_dataset_from_selfplay(game_data, params)
 
   # Hold out 5% of games for evaluation.
   if random.random() < params.holdout_pct:
     fname = os.path.join(
-        holdout_dir, ('{}'+_TF_RECORD_SUFFIX).format(output_name))
+        selfplay_dirs['holdout_dir'], ('{}' + _TF_RECORD_SUFFIX).format(
+            output_name))
```

Comment:
Seems to be a simple string concat, let's get rid of format().

Reply:
Makes sense.